docs/manual-curation/step2-manual-curation.md

```diff
@@ -3,7 +3,7 @@
 The goals of the manual curation:
 * All traits which are linked to NT expansion (nucleotide repeat expansion) variants must be curated. Those are marked as "NT expansion" in the “Notes” column.
 * All traits with occurrence ≥ **10** must be curated. Additionally, if there are less than **200** such terms, then the top 200 terms must be curated.
-* _Suggested previous mapping_ traits should be checked for any terms that have become obsolete since the last iteration. This can be done by filtering then searching for the string EFO\_OBSOLETE
+* _Suggested previous mapping_ traits should be checked for any terms that have become obsolete since the last iteration. These will be colored red and have a _suggested replacement mapping_ provided in the appropriate column. If no replacement is provided, curate as usual.
```

Comment:
I assume it can be that there are obsolete terms (i.e. in red) that have no replacement.
```suggestion
* _Suggested previous mapping_ traits should be checked for any terms that have become obsolete since the last iteration. These will be colored red and likely have a _suggested replacement mapping_ provided in the appropriate column. If no replacement is provided, curate as usual.
```